test/mysql/TestDBInit.chpl

```diff
@@ -0,0 +1,24 @@
+module Main {
```

Comment:
Are you able to use this method from other files?

Reply:
I'm a little unclear about what you mean - is about the main() method in this module? If yes, I'll have to see if it is accessible from some other file (although I don't see a purpose for doing that, would help if you could elaborate a bit about it, thanks!)

Reply:
This can be ignored for now.